webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareConfig.js

```diff
@@ -0,0 +1,188 @@
+      statusSelector: state => selectRPMPackagesComparisonStatus(state, versionOneId, versionTwoId),
+      autocompleteEndpoint: '/packages/auto_complete_search',
+      fetchItems: params => getRPMPackagesComparison(
+        versionOneId,
+        versionTwoId,
+        params,
+      ),
+      columnHeaders: [
+        {
+          title: __('Name'),
+          getProperty: item => item?.nvrea,
+        },
+        { title: __(`Version ${versionOne}`), getProperty: item => compareContent(item, versionOneId) },
+        { title: __(`Version ${versionTwo}`), getProperty: item => compareContent(item, versionTwoId) },
+      ],
+    },
+    {
+      name: __('RPM Package Groups'),
+      getCountKey: (itemVersionOne, itemVersionTwo) =>
+        itemVersionOne?.package_group_count || itemVersionTwo?.package_group_count,
+      responseSelector: state => selectPackageGroupsComparison(state, versionOneId, versionTwoId),
+      statusSelector: state =>
+        selectPackageGroupsComparisonStatus(state, versionOneId, versionTwoId),
+      autocompleteEndpoint: '/package_groups/auto_complete_search',
+      fetchItems: params => getPackageGroupsComparison(versionOneId, versionTwoId, params),
+      columnHeaders: [
+        { title: __('Name'), getProperty: item => item?.name },
+        { title: __(`Version ${versionOne}`), getProperty: item => compareContent(item, versionOneId) },
+        { title: __(`Version ${versionTwo}`), getProperty: item => compareContent(item, versionTwoId) },
+      ],
+    },
+    {
+      name: __('Files'),
+      getCountKey: (itemVersionOne, itemVersionTwo) =>
+        itemVersionOne?.file_count || itemVersionTwo?.file_count,
+      responseSelector: state => selectFilesComparison(state, versionOneId, versionTwoId),
+      statusSelector: state => selectFilesComparisonStatus(state, versionOneId, versionTwoId),
+      autocompleteEndpoint: '/files/auto_complete_search',
+      fetchItems: params => getFilesComparison(versionOneId, versionTwoId, params),
+      columnHeaders: [
+        {
+          title: __('Name'),
+          getProperty: item => item?.name,
+        },
+        { title: __(`Version ${versionOne}`), getProperty: item => compareContent(item, versionOneId) },
+        { title: __(`Version ${versionTwo}`), getProperty: item => compareContent(item, versionTwoId) },
+      ],
+    },
+    {
+      name: __('Errata'),
+      getCountKey: (itemVersionOne, itemVersionTwo) =>
+        itemVersionOne?.erratum_count || itemVersionTwo?.erratum_count,
+      responseSelector: state => selectErrataComparison(state, versionOneId, versionTwoId),
+      statusSelector: state => selectErrataComparisonStatus(state, versionOneId, versionTwoId),
+      autocompleteEndpoint: '/errata/auto_complete_search',
+      fetchItems: params => getErrataComparison(versionOneId, versionTwoId, params),
+      columnHeaders: [
+        { title: __('Name'), getProperty: item => item?.name },
```

Comment:
These columns should match the columns in version details config for this and other content types..